packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -329,11 +333,17 @@ export class DocumentDeltaConnection
 	/**
 	 * Submits a new signal to the server
 	 *
-	 * @param message - signal to submit
+	 * @param content - Content of the signal.
+	 * @param targetClientId - When specified, the signal is only sent to the provided client id.
 	 */
-	public submitSignal(message: IDocumentMessage): void {
+	public submitSignal(content: unknown, targetClientId?: string): void {
 		this.checkNotDisposed();
-		this.emitMessages("submitSignal", [[message]]);
+
+		if (targetClientId) {
+			throw new Error("Sending signals to specific client ids is not supported");
+		}
+
+		this.emitMessages("submitSignal", [[content]]);
```

Comment:
so this in going the unknown[][] route, shouldn't it be ISentSignalMessage[]

Reply:
I removed the typing changes from this PR